ports/ziiaol/Zelda II Enhanced.sh

```diff
@@ -42,23 +41,27 @@ if [ ! -f patchlog.txt ]; then
         source "$controlfolder/utils/patcher.txt"
         $ESUDO kill -9 $(pidof gptokeyb)
     else
-        echo "This port requires the latest version of PortMaster." > $CUR_TTY
+        echo "This port requires the latest version of PortMaster."
```

Comment:
pm_message ?

Reply:
Not for these two, these are meant to be a part of the log.txt and I don't think `pm_message` also outputs to log.